### PR TITLE
Added Signup page test automation 

### PR DESCRIPTION
**Pull request by iwazowsky**

### CODE REVIEW

File: cypress/cypress/actions/signup.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
+import SignupPage from '../integration/pom/SignupPage';
+
+const signup = (userType, {
+  phoneNumber = '', password = '', passwordConfirm = password, firstName = '', lastName = '', email = '', carRegistrationNumber = '', carManufacturer = '', carModel = '', imgPath = '',
```
Comment (ArtyomKosakyan) [inline cypress/cypress/actions/signup.js:5]: Yo can avoid lines like this (very big lines)

File: cypress/cypress/support/commands.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (ArtyomKosakyan) [inline cypress/cypress/support/commands.js:1]: Why disabled?

**Review by ArtyomKosakyan — APPROVED** [review]
I have no important comments.

File: cypress/cypress/support/commands.js (hunk 2 above)
Comment (iwazowsky) [reply cypress/cypress/support/commands.js:1]: there was a bug on cypress directory

File: cypress/cypress/actions/signup.js (hunk 1 above)
Comment (iwazowsky) [reply cypress/cypress/actions/signup.js:5]: yeah,  should have put them on separate lines 😄

**Pull request merged**